test/functional/utilities/ctrclient.go

```diff
@@ -0,0 +1,46 @@
+package testutilities
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+// Global options for connecting to ctr.exe
+// Flags are passed to parent module, cannot import them here without causing circular dependencies
+// todo: restructure `LayerFolders` and `CreateWCOWUVM*` functions to use CtrClientOptions
+// or move `utilities/*` into parent path, similar to `tests/cri-containerd`
+
+type CtrClientOptions struct {
+	Path      string
+	Address   string
+	Namespace string
+}
+
+var ctro CtrClientOptions
+
+func GetCtrClientOptions() *CtrClientOptions {
```

Comment:
maybe we should remove these changes since you already have another PR open to fix functional tests?